src/modules/settings/index.tsx

```diff
@@ -38,6 +39,31 @@ export default class Settings {
       return undefined;
     }
   }
+
+  async appSetting(name: string): Promise<any | undefined> {
```

Comment:
here you need to specify the type that the function returns when it is not undefined